compiler/optimizer/CompactLocals.cpp

```diff
@@ -458,32 +460,29 @@ TR_CompactLocals::createInterferenceBetween(TR_BitVector *bv)
          {
          i2 = wbvi.getNextElement();
 
-         sym1 = (*_localIndexToSymbolMap)[i1];
-         sym2 = (*_localIndexToSymbolMap)[i2];
+         ig1 = (*_localIndexToIGNode)[i1];
+         ig2 = (*_localIndexToIGNode)[i2];
 
-         if (sym1 && sym2 &&
-             eligibleLocal(sym1) && eligibleLocal(sym2) &&
-             !_localsIG->hasInterference(sym1, sym2))
+         if (ig1 && ig2)
```

Comment:
The notion of an eligibleLocal is no longer required ? Just checking because the code used to check for that but does not do so anymore with the change having been done.

Reply:
This is safe as `_localIndexToIGNode` will only receive an entry for a symbol if it is an `eligibleLocal`. Moreover, whether it is an `eligibleLocal` will not change throughout this analysis.